packages/vue-components/src/panels/PanelBase.js

```diff
@@ -135,25 +135,17 @@ export default {
       }
 
       if (this.localExpanded) {
-        /*
-          Collapse panel.
-          Panel's maxHeight is 'none' at the moment, as event listener set it to 'none' after expansion.
-          Thus, we need to reset the maxHeight to its current height for collapse transition to work.
-        */
-        this.$refs.panel.style.maxHeight = `${this.$refs.panel.scrollHeight}px`;
-
         requestAnimationFrame(() => {
           // To enable behaviour of auto window scrolling during panel collapse
           if (this.$el.getBoundingClientRect().top < 0) {
             const headerHeight = jQuery('header[sticky]').height() || 0;
-            jQuery('html').animate({
-              scrollTop: window.scrollY + this.$el.getBoundingClientRect().top - headerHeight - 3,
-            }, 500, 'swing');
+            window.scrollTo(0, window.scrollY + this.$el.getBoundingClientRect().top - headerHeight - 3);
```

Comment:
It looks much better than the jQuery one, but oddly it still animates, contrary to the mdn documentation.. Is that intended?

Reply:
Hi @ang-zeyu, could I check what do you mean by it still animates? 
It seems to not be animated (except for the buttons) on my end

https://user-images.githubusercontent.com/77204346/221115277-dd68aacd-2e9d-4eab-a556-32b167ec2c09.mp4

Reply:
This line ^ (`window.scrollTo`)

Reply:
![paneltransition](https://user-images.githubusercontent.com/3306138/221340159-7eafea10-3812-4890-95a9-57995c5eac16.gif)

Reply:
Ah I see! 
@ang-zeyu is this the desired behaviour? 

https://user-images.githubusercontent.com/77204346/221341761-4f7d6a23-20b2-4d75-960d-1883cbe78976.mp4

Reply:
Yes